packages/astro/src/@types/astro.ts

```diff
@@ -1953,50 +1953,7 @@ export interface AstroUserConfig {
 		 * In the event of route collisions, where two routes of equal route priority attempt to build the same URL, Astro will log a warning identifying the conflicting routes.
 		 */
 		globalRoutePriority?: boolean;
-
-		/**
-		 * @docs
```

Comment:
No issues, since we're entirely deleting docs here!  :smile: 

I'm assuming we'll need a corresponding PR to docs [in the existing domains section](https://docs.astro.build/en/guides/internationalization/#domains-experimental) that:

- removes "experimental" from the section heading
- changes the `<Since />` component to `v="4.9.0"`
- updates the code sample to remove the experimental flag
- moves this entire section up to the `routing` section (probably before "manual", which sounds like it should come last?)

Otherwise, it seems like there are no changes to actual usage here?

Reply:
Thank you, I will make a PR tomorrow